requirements/common.in

```diff
@@ -3,7 +3,7 @@
 # and requirements/prod.txt.
 # See requirements/README.md for more detail.
 # Django itself
-Django==1.11.*  # https://github.com/zulip/zulip/issues/10835
+Django==2.2.8
```

Comment:
Should be `Django==2.2.*` since we don’t want to block point releases. (Upstream is up to 2.2.10.)